### ISSUE TITLE
How to visual Fig. 4 in your paper

### ISSUE THREAD

**Issue by Elbenhorn** [ma-xu/pointMLP-pytorch#83]
Thank you for your amazing work!
Could you share the code of drawing Fig.4 (Loss landscape along two rand directions.)

**Comment by ma-xu** [comment]
I will send it to you by the end of May (or early June). Sry that I'm really busy recently.

**Comment by LiuHahahaha** [comment]
I also have the same request. Can you share the code?

**Comment by ma-xu** [comment]
@Elbenhorn @LiuHahahaha Sorry about the late reply. 

We firstly get the raw data by [lossland.py](https://drive.google.com/file/d/1bZL_ZRcfaJBWwrxnbajATBTCW6QhxsAI/view?usp=sharing), then we plot the figure by [plot.py](https://drive.google.com/file/d/1_ygidzgere1r-wLthb9O2Rffp_FKba95/view?usp=sharing). Note that both scripts are not cleaned andn you may need to change some settiings accordingly (e.g., checkpoint paths, ranges, etc.). More details can be found here: Li, Hao, et al. "Visualizing the loss landscape of neural nets." NeurIPS 2018.

Also in issue: https://github.com/ma-xu/pointMLP-pytorch/issues/42

**Comment by Elbenhorn** [comment]
Very appreciate it!
